test/spec/api/v4/filter/bounding-box-leaflet.spec.js

```diff
@@ -0,0 +1,11 @@
+var carto = require('../../../../../src/api/v4/index');
+
+describe('api/v4/filter/bounding-box-leaflet', function () {
+  describe('constructor', function () {
+    it('should throw a descriptive error when initialized with invalid parameters', function () {
+      expect(function () {
```

Comment:
Add more tests including bool, number, string, etc as map

Reply:
I dont think is a good idea, the essence of the test is "throw an error when parameter is not a map?"

All those cases are equivalent

Reply:
That's true.